editor/script_create_dialog.cpp

```diff
@@ -330,8 +332,13 @@ void ScriptCreateDialog::_template_changed(int p_template) {
 			dic_templates_project[parent_name->get_text()] = sinfo.get_hash();
 			EditorSettings::get_singleton()->set_project_metadata("script_setup", "templates_dictionary", dic_templates_project);
 		} else {
-			// Save template info to editor dictionary (not a project template).
-			templates_dictionary[parent_name->get_text()] = sinfo.get_hash();
+			// Save template into to editor dictionary (not a project template).
```

Comment:
The "info" wasn't a typo here I think. At least the new sentence doesn't make sense ("into to").

Reply:
That's the original comment, I'll fix it.